<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBook-Pro-de-Alexis-3.local" name="fr.eseo.atribus.controller.CompetencesControllerTest" tests="8" failures="1" timestamp="27 May 2016 12:54:09 GMT" time="0.764" errors="0">
  <testcase name="afficherAjouterCompetences" time="0.617" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="ajouterCompetencePost" time="0.014" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="ajouterUneCompetenceGet" time="0.002" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="afficherAssocierCompetencesUe" time="0.043" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="associerCompetenceUe" time="0.025" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="modifierCompetenceGet" time="0.004" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="afficherModifierAssocierCompetencesUe" time="0.051" classname="fr.eseo.atribus.controller.CompetencesControllerTest"/>
  <testcase name="modifierCompetencePost1" time="0.008" classname="fr.eseo.atribus.controller.CompetencesControllerTest">
    <failure type="java.lang.AssertionError" message="Status expected:&lt;200&gt; but was:&lt;400&gt;">
      <![CDATA[java.lang.AssertionError: Status expected:<200> but was:<400>
	at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:60)
	at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:89)
	at org.springframework.test.web.servlet.result.StatusResultMatchers$10.match(StatusResultMatchers.java:655)
	at org.springframework.test.web.servlet.MockMvc$1.andExpect(MockMvc.java:171)
	at fr.eseo.atribus.controller.CompetencesControllerTest.modifierCompetencePost1(CompetencesControllerTest.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </failure>
  </testcase> <!-- modifierCompetencePost1 -->
</testsuite> <!-- fr.eseo.atribus.controller.CompetencesControllerTest -->
